STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/src/az_ulib_dm.c

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This is also auto-generated, correct?

Reply:
I think for now they're added by hand

Reply:
No, this is a fake implementation of the Device Manager. In the future this code will be responsible to download and install packages from the cloud.